cmd/helm/flags.go

```diff
@@ -31,7 +31,7 @@ import (
 const outputFlag = "output"
 
 func addValueOptionsFlags(f *pflag.FlagSet, v *values.Options) {
-	f.StringSliceVarP(&v.ValueFiles, "values", "f", []string{}, "specify values in a YAML file or a URL(can specify multiple)")
+	f.StringSliceVarP(&v.ValuesFiles, "values", "f", []string{}, "specify values in a YAML file or a URL(can specify multiple)")
```

Comment:
This is more consistent with `valuesfile(s)` elsewhere, along with `values` in this line.